automation/run-tests.sh

```diff
@@ -115,10 +100,10 @@ function run_tests {
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_UNIT_PY38 ];then
         if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running unit test in $CONTAINER_IMAGE container is not " \
+            echo "Running python 3.7 unit test in $CONTAINER_IMAGE container i not " \
```

Comment:
There is a typo, it now says "i not" instead of "is not".

Reply:
Also, this should be Python 3.8

Reply:
Done, thanks!